test/gui/suite.conf

```diff
@@ -4,4 +4,5 @@ HOOK_SUB_PROCESSES=false
 IMPLICITAUTSTART=false
 LANGUAGE=Python
 OBJECTMAPSTYLE=script
+VERSION=3
```

Comment:
This is needed for Squish IDE to generate UI selectors as an object. Previously, it got removed in one of the refactoring PRs.